share/revng/test/configuration/revng-qa/for-simplify-switch.yml

```diff
@@ -0,0 +1,22 @@
+tags:
+  - name: for-simplify-switch
+  - name: for-analysis
+    variables:
+      GCC_CFLAGS:
+        - -O2
+        - -fno-stack-protector
+        - -fomit-frame-pointer
+        - -fno-reorder-functions
+        - -fno-unwind-tables
+        - -fno-asynchronous-unwind-tables
+        - -fno-stack-check
+        - -fno-optimize-sibling-calls
+        - -fno-inline-functions
+        - -fno-inline-small-functions
+        - -fno-align-functions
+        - -fno-optimize-sibling-calls
```

Comment:
These make no sense here. We're just invoking the assembler, don't blindly copy paste!